content/docs/command-reference/exp/show.md

```diff
@@ -0,0 +1,86 @@
+# exp show
+
+Display `dvc experiments` as a table, with optional formatting.
+
+> Press `q` to exit.
+
+## Synopsis
+
+```usage
+usage: dvc exp show [-h] [-q | -v] [-a] [-T] [--all-commits] [--no-pager]
+                    [--include-metrics <metrics_list>]
+                    [--exclude-metrics <metrics_list>]
+                    [--include-params <params_list>]
+                    [--exclude-params <params_list>]
+                    [--sort-by <metric/param>] [--sort-order {asc,desc}]
+                    [--no-timestamp] [--sha] [--show-json]
+                    [--precision <n>]
+```
+
+## Description
+
+Enters a screen that shows all of the relevant experiments you've run. Typically
+you can exit this screen by typing `q` in your keyboard.
```

Comment:
```suggestion
Shows all of the relevant experiments you've run. By default, enters a screen that
you can typically exit by typing `q` in your keyboard (or use `--no-pager` to print
output in your terminal without entering this screen).
```
This probably needs some work. The point is that we mention the default option here using very plain language without referring to the `--no-pager` option, and the `--no-pager` description doesn't use similar plain language.

Reply:
Thanks! Def better text, Using something very similar to this now. I'll review all the options text next (hopefully that finishes this iteration).